src/main/java/com/example/webapp/dto/EmployeeDTO.java

```diff
@@ -0,0 +1,18 @@
+package com.example.webapp.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+public class EmployeeDTO {
+    private long id;
```

Comment:
Better to use wrapper classes: Integer, Long. They don't have default values: for example if you'll not declare salary by default it will be set as 0.00

Reply:
Done